### ISSUE TITLE
`do_python.py` training is classification only

### ISSUE THREAD

**Issue by vatai** [undertherain/benchmarker#44]
# Problem
The training for pytorch has classification i.e. cross entropy loss hardwired - see [this line](https://github.com/undertherain/benchmarker/blob/87a2c1bcf3ffdd6f102860871c848127260ba3bb/benchmarker/modules/do_pytorch.py#L29).
# Proposed solution
The `self.net` member of `INeuralNet` stores the model returned by `get_kernel()` of the problem (e.g. `get_kernel()` defined in `pytorch.py`). Instead of putting just the model in `self.net`, we could add the model and the loss as well... i.e. so all the pytorch `get_kernel` functions should `return (model, loss)` and `do_python.py` should use `self.net[0]` instead of `self.net` and `self.net[1]` instead of `CrossEntropyLoss()`

**Comment by undertherain** [comment]
this is already covered in #31

**Comment by vatai** [comment]
Copied Proposed solution.
